### DISCUSSION TITLE
Off to a bad start

### DISCUSSION THREAD

**Discussion by thetechnodino** [Zettlr/Zettlr#3753] [General]
I am using Zettlr version: 2.3.0 on windows 11 (64 bit) and I am off to a rough start.
When I open the preferences dialog (use the cog, the keyboard or the menu item) I see only 1 tab  (screen shot attached).

Restart, power off restart, reinstall and all the troubleshooting hints and problem persists.  User error?
![Preferences_2022-09-06 204027](https://user-images.githubusercontent.com/47829077/188770669-6cb59589-e4fb-401e-a202-d1dd7df18325.png)

Any help appreciated.
Thanks
John

**Comment by nathanlesage** [comment]
It is difficult to judge what is happening from a cropped screenshot, so I'm just going to give you general advice: you can move the window using the Windows key plus arrow keys, or by right clicking the window in the taskbar and selecting "Move". This should enable you to bring the window bar into view.

**Comment by thetechnodino** [comment]
Thanks. When I moved the screen, the bar appeared.

---
John Payne

630-815-2989 | www.pde-usa.net [3]

Sola Deo Gloria

On 2022-09-07 06:58, Hendrik Erz wrote:

> It is difficult to judge what is happening from a cropped screenshot, 
> so I'm just going to give you general advice: you can move the window 
> using the Windows key plus arrow keys, or by right clicking the window 
> in the taskbar and selecting "Move". This should enable you to bring 
> the window bar into view.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/Zettlr/Zettlr/discussions/3753#discussioncomment-3583864
[2] 
https://github.com/notifications/unsubscribe-auth/ALM5AVLDTQULLNN4METAIRDV5BYMHANCNFSM6AAAAAAQGKN5PM
[3] http://www.pde-usa.net

**Comment by thetechnodino** [comment]
Thanks for the suggestion; it worked.

---
John Payne

630-815-2989 | www.pde-usa.net [3]

Sola Deo Gloria

On 2022-09-07 06:58, Hendrik Erz wrote:

> It is difficult to judge what is happening from a cropped screenshot, 
> so I'm just going to give you general advice: you can move the window 
> using the Windows key plus arrow keys, or by right clicking the window 
> in the taskbar and selecting "Move". This should enable you to bring 
> the window bar into view.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/Zettlr/Zettlr/discussions/3753#discussioncomment-3583864
[2] 
https://github.com/notifications/unsubscribe-auth/ALM5AVLDTQULLNN4METAIRDV5BYMHANCNFSM6AAAAAAQGKN5PM
[3] http://www.pde-usa.net
